package.json

```diff
@@ -29,8 +29,56 @@
         "type": "git",
         "url": "https://github.com/shader-slang/slang-vscode-extension"
     },
-    "main": "./client/out/main",
+    "main": "./client/dist/nativeClientMain",
+    "browser": "./client/dist/browserClientMain",
     "contributes": {
+        "commands": [
+            {
+                "command": "slang.compile",
+                "title": "Slang Compile",
+                "category": "Slang",
+                "enablement": "editorLangId == slang",
```

Comment:
default hotkey this to `Ctrl+B`?

Reply:
There are currently default hotkeys for run and compile:
```json
{
    "command": "slang.compile",
    "key": "ctrl+shift+b",
    "when": "editorLangId == slang"
},
{
    "command": "slang.playgroundRun",
    "key": "ctrl+shift+r",
    "when": "editorLangId == slang"
}
```
Do you want them changed to `f5` and `Ctrl+B`?